.env

```diff
@@ -12,6 +12,7 @@ PROCESSOR_PROMETHEUS_PORT=3337
 GQL_PORT=4350
 
 # Archive gateway url
+CUSTOM_ARCHIVE_GATEWAY_URL=https://joystream.archive.subsquid.io/graphql
```

Comment:
I don't think this should be commited, for development purposes it's better to use the default `http://localhost:8888/graphql`, it can be changed on demand via a command line if needed
```suggestion
```